bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/internal/models/v1/TeaserImpl.java

```diff
@@ -99,21 +100,27 @@
     @PostConstruct
     private void initModel() {
         withCTA = properties.get(Teaser.PN_WITH_CTA, withCTA);
```

Comment:
I would also call this something like `actionsEnabled` or we're sticking with CTA `ctasEnabled`.